DecipherNow/gm-fabric-dashboard
# WIP: 1348 react router redux

Resolves #1348 

## Proposed Changes

  - Refactor app to not use react-router-redux
  - Refactor old non-sds behavior from AppHeader
  - Refactor out old unused props in Main
  
## How to Test

  - npm test to see if stuff blows up
  - Run app and click around to see if things work as expected.
